packages/gatsby-source-contentful/src/__tests__/__snapshots__/extend-node-type.js.snap

```diff
@@ -41,7 +41,7 @@ Object {
 exports[`contentful extend node type resolveResponsiveResolution generates responsive resolution data for images using all options 1`] = `
 Object {
   "aspectRatio": 1.1278195488721805,
-  "base64": "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQEAlgCWAAD/2wBDAAMCAgMCAgMDAwMEAwMEBQgFBQQEBQoHBwYIDAoMDAsKCwsNDhIQDQ4RDgsLEBYQERMUFRUVDA8XGBYUGBIUFRT/2wBDAQMEBAUEBQkFBQkUDQsNFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBT/wAARCAAbABQDAREAAhEBAxEB/8QAGQABAAIDAAAAAAAAAAAAAAAABgIHAwQJ/8QALhAAAQMDAgIHCQAAAAAAAAAAAQIDBAAFEQYSEzEHFCEiQVFhFRYkMkJDcXKj/8QAGQEAAgMBAAAAAAAAAAAAAAAAAQIAAwQF/8QAJREAAQQCAQEJAAAAAAAAAAAAAQACAxEEQSEiEhMxQlFhcYGh/9oADAMBAAIRAxEAPwDp4Zz3vV1Tf8OIXF2YHzb8Zzz5UNpL6qWixdparPY3lO5clSUIdVtHeSQo49OQoXwl7RoFJaZWo8tWNcqHh7Mz/U0u0nnRF2+O3DRmiJtilRJLDtyY4jzm4oLQDgXtx9WRjyrSGNhcWZAINfuvpcl078mCOXBc0guFk3VAkOqtqz6oXZQK7amhQ+lJy0rcUJy9PrlIRtOCgOqBOeXMVZ3TywzaHCwuyo2ZTccnqLSR8A+qPR2o9v0nYLbGQhhqM68oIbSAEYX4Afuazue6TlxspYo2RRNjjAAF+CtiI+JUVl4fcQFdnqKsW8cqKrfGXN62plJk8Is8Qjt2E52/jNH2QoXamIrIxhlsY5d0dlBMsgAAwBgeQqKL/9k=",
+  "baseUrl": "//images.contentful.com/ubriaw6jfhm1/10TkaLheGeQG6qQGqWYqUI/5421d3108cbb699561acabd594fa2cb0/ryugj83mqwa1asojwtwb.jpg",
```

Comment:
We need a base64 *string* not a link so that we can embed the base64 string directly in the HTML so it can show immediately upon loading the page.

Reply:
@KyleAMathews It's actually a different property; `base64` has been removed, and `baseUrl` is something I started returning to be used in the `getBase64Image`. It's not a graphql field so it will not be exposed to users.

For example `resolveResponsiveResolution` returns it here: https://github.com/gatsbyjs/gatsby/pull/3617/files#diff-dc42475a9f0ed45d6e1bf554ec9958a2R199

And then `getBase64Image` uses it: https://github.com/gatsbyjs/gatsby/pull/3617/files#diff-dc42475a9f0ed45d6e1bf554ec9958a2R78

Just double-checked that on my branch queries return base64 strings as expected:

![screenshot 2018-01-22 14 23 57](https://user-images.githubusercontent.com/11855531/35240099-5e5a17c6-ff80-11e7-853a-004c9e565a1f.png)

Reply:
Oooo gotcha! Ok :-) totally makes sense. Didn't look closely, clearly, at what that snapshot was for!